R/get-non-catalogue-data.R

```diff
@@ -164,34 +170,11 @@ fsa <- function(class = 'sf', ask = interactive(), force = FALSE) {
 #' }
 #'
 #' @export
-mapsheets_250K <- function(class = 'sf', ask = interactive(), force = FALSE) {
-  dir <- data_dir()
-  fpath <- file.path(dir, "mapsheets_250K.rds")
+mapsheets_250K <- function(class = 'sf') {
+
+
+  ret <- bcmaps::mapsheets_250K_data
```

Comment:
I don't think you want/need to add the namespace prefix here...?

Reply:
For some reason, the tests couldn't find it unless I did it this way. Open to alternative suggestions.